### ISSUE TITLE
Question: How to override editor module config?

### ISSUE THREAD

**Issue by nixmaniack** [sorin-ionescu/prezto#1236]
`editor` module redefines `v` key for `vicmd` keymap which in default config is `visual-mode`. As per [this readme](https://github.com/sorin-ionescu/prezto/blob/4f19700919c8ebbaf75755fc0d03716d13183f49/runcoms/README.md#file-descriptions) there's no way I can redefine it to default keybinding. 

The only place keybindings should go is `~/.zshrc` which is sourced before `~/.zpreztorc` so there doesn't seem to be a right place to override this.
I even tried adding at the end of `~/.zpreztorc` but wasn't successful in rebinding.

**Comment by fuadsaud** [comment]
`~/.zshrc` should be loading `~/.zprezto/init.sh`, which, in turn, loads `~/.zpreztorc`. Did you try adding `bindkey -M vicmd "v" foobar` to the end of your `~/.zshrc`?

**Comment by nixmaniack** [comment]
@fuadsaud Yes, I tried already - doesn't work.  You're right that `~/.zpreztorc` is loaded after `~/.zshrc` - the readme I linked to states the same. 

The issue is - `editor` module [resets keybindings](https://github.com/sorin-ionescu/prezto/blob/4f19700919c8ebbaf75755fc0d03716d13183f49/modules/editor/init.zsh#L205) which I set in `~/.zshrc` hence there's no other place to rebind the keys in standard way. 

Right now, dirty solution is to comment what is not required from `editor` module and add what is needed there - which is not what I want.
